python/qpid_dispatch/management/qdrouter.json

```diff
@@ -1209,7 +1209,12 @@
                 "cost": {
                     "description": "Reachability cost",
                     "type": "integer"
+                },
+                "lastTopoChange": {
+                     "description": "Timestamp showing the most recent change to this node's neighborhood.",
+                     "type": "string"
```

Comment:
This should be an integer, not a string.  Seconds since the epoch is more than sufficient granularity.